### ISSUE TITLE
`matplotlib` won't display correctly 

### ISSUE THREAD

**Issue by mvesin** [fedbiomed/fedbiomed#1176]
With current version of `matplotlib` in FB environment (node or researcher) in `develop` branch, this won't display correctly (while it did until recently). Remove/redo the conda env to trigger the issue.

```
import numpy as np
import matplotlib.pyplot as plt

plt.imshow(np.array([[1, 2,], [2, 1]]))
plt.show()
```

Error message on the console:
```
Exception in Tkinter callback
Traceback (most recent call last):
...
ValueError: object __array__ method not producing an array
```

Looks like a bug introduced in newer version of (?)matplotlib.
numpy and matplotlib are automatically pulled by FB envs entries, so we don't directly control the versions (the pulling packages should ensure coherence ...)

**Comment by mvesin** [comment]
Looks more like a matplotlib issue as this one not using numpy also fails (while it works with other versions of matplotlib):

```
import matplotlib.pyplot as plt

plt.imshow([[1, 2,], [2, 1]])
plt.show()
```

**Comment by ybouilla** [comment]
Fixed by merging #1178
